solr/test-framework/src/java/org/apache/solr/SolrJettyTestBase.java

```diff
@@ -74,16 +73,23 @@ public static JettySolrRunner createAndStartJetty(
             .withServlets(extraServlets)
             .withSSLConfig(sslConfig.buildServerSSLConfig())
             .build();
-
+    ;
     Properties nodeProps = new Properties();
-    if (configFile != null) nodeProps.setProperty("solrconfig", configFile);
```

Comment:
You removed setting "solrconfig" but kept "schema" on the line below.  Doesn't make sense to me; either both or neither.  I suspect you can remove both since these are supplied when we create the collection explicitly below.